packages/server/src/api/controllers/row/index.ts

```diff
@@ -100,25 +100,24 @@ export async function find(ctx: any) {
   })
 }
 
-export async function destroy(ctx: any) {
+export async function destroy(ctx: UserCtx<DeleteRowRequest>) {
   const appId = ctx.appId
-  const inputs = ctx.request.body
+  const inputs: DeleteRowRequest = ctx.request.body
```

Comment:
Can do without the :`DeleteRowRequest` definition here, it'll automatically be inferred via the usage in the function signature